javalin/src/main/java/io/javalin/config/PluginConfig.kt

```diff
@@ -33,6 +33,11 @@ class PluginConfig(private val pvt: PrivateConfig) {
     fun enableRedirectToLowercasePaths() = register(RedirectToLowercasePathPlugin())
     fun enableBasicAuth(username: String, password: String) = register(BasicAuthPlugin(username, password))
     fun enableSslRedirects() = register(SslRedirectPlugin())
-    fun enableCors(userConfig: Consumer<CorsPluginConfig>) = register(CorsPlugin(userConfig))
+    fun enableCors(userConfig: Consumer<CorsPluginConfig>) {
+        if (pvt.cors == null) {
+            pvt.cors = CorsContainerPlugin().also { register(it) }
+        }
+        pvt.cors?.addCors(userConfig)
```

Comment:
I'm wondering if we could keep this as an app-attribute instead, or manage entirely through the plugin 🤔

Reply:
With the switch to allowing multiple configurations this is not necessary anymore.